.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -12,11 +12,11 @@ jobs:
     matrix:
       win_c_compilervs2015cxx_compilervs2015python3.6:
         CONFIG: win_c_compilervs2015cxx_compilervs2015python3.6
-        CONDA_BLD_PATH: C:\\bld\\
```

Comment:
@ocefpaf why is this always changing from Drive C to D? It's the re-rederer that changes this, do we need to change it back to C again?

Reply:
We need to fix `conda-smithy` to always use `C:` instead of `D:`. (And I keep forgetting to bring that up, sorry.) Let's change it manually for now.